src/main/drivers/dshot.c

```diff
@@ -330,6 +325,14 @@ uint32_t getDshotAverageRpm(void)
 
 #endif // USE_DSHOT_TELEMETRY
 
+// Used with serial esc telem as well as dshot telem
```

Comment:
Moving this outside of the ifdef USE_DSHOT_TELEMETRY seems to have caused the breaking in test. The error message reports "undefined reference to `motorConfig_System'" which I think comes from the PG macro. I have no idea why that breaks in test

Reply:
I've wrapped the moved function in a new #if block, hoping this resolves the problem